src/topology/uniform_space/absolute_value.lean

```diff
@@ -0,0 +1,73 @@
+/-
+Copyright (c) 2019 Patrick Massot. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Patrick Massot
+-/
+
+import data.real.cau_seq
+import topology.uniform_space.basic
+
+/-!
+# Uniform structure induced by an absolute value
+
+We build an uniform space structure on a commutative ring `R` equipped with an absolute value into
+a linear ordered field `𝕜`. Of course in the case `R` is `ℚ`, `ℝ` or `ℂ` and
+`𝕜 = ℝ`, we get the same thing as the metric space construction, and the general construction
+follows exactly the same path.
+
+Note that we import `data.real.cau_seq` because this is where absolute values are defined, but
```

Comment:
```suggestion
## Implementation details

Note that we import `data.real.cau_seq` because this is where absolute values are defined, but
```
Maybe add this header?